test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/Package/PackageModelTests.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test.Models
+{
+    public class PackageModelTests
+    {
+        [Fact]
+        public void PackageModelCtr_IdAndVersion_ReturnsValueFromIdentity()
```

Comment:
Nit: This method is actually testing the return value of two related, but distinct properties, not the constructor. Consider breaking this into two tests and name them according to which getters are tested. Note, you can share the same package identity arrangement between both tests by using a setup / constructor for this test class.